src/components/organisms/HotKeysHandler/HotKeysHandler.tsx

```diff
@@ -95,7 +97,8 @@ const HotKeysHandler = () => {
   );
 
   useHotkeys(hotkeys.TOGGLE_SETTINGS.key, () => {
-    dispatch(toggleSettings());
+    dispatch(setActiveSettingsPanel(SettingsPanel.ActiveProjectSettings));
```

Comment:
Why should we select the active settings pane here? What is the first panel inside settings that is selected when you open it?